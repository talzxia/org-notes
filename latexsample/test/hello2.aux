\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\HyPL@Entry{2<</S/D>>}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}泰山职业技术学院}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}信息技术工程系}{1}{section.1.1}}
\newlabel{LastPage}{{}{1}{}{page.1}{}}
\xdef\lastpage@lastpage{1}
\xdef\lastpage@lastpageHy{1}
