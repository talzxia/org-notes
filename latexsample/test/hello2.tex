% hello, world
% 编译改为xelatex
\documentclass[color=cyan,mathpazo,titlestyle=hang]{elegantbook}

\author{ddswhu \& LiamHuang0205}
\email{elegantlatex2e@gmail.com}
\zhtitle{优美的\LaTeX{} 书籍}
\zhend{模板}
\entitle{Elegant\LaTeX{} Book}
\enend{Template}
\version{2.00}
\myquote{泰山职业技术学院————信息技术工程系}
\logo{logo1.pdf}
\cover{cover.pdf}

% green color
\definecolor{main1}{RGB}{0,120,2}
\definecolor{seco1}{RGB}{230,90,7}
\definecolor{thid1}{RGB}{0,160,152}
% cyan color
\definecolor{main2}{RGB}{0,175,152}
\definecolor{seco2}{RGB}{239,126,30}
\definecolor{thid2}{RGB}{120,8,13}
% blue color
\definecolor{main3}{RGB}{20,50,104}
\definecolor{seco3}{RGB}{180,50,131}
\definecolor{thid3}{RGB}{7,127,128}

\usepackage{makecell}
\usepackage{lipsum}
\usepackage{texnames}
\usepackage{fancyvrb}
\setCJKmainfont{方正小标宋简体}

\begin{document}
\maketitle
\tableofcontents
\mainmatter



\chapter{泰山职业技术学院}
\section{信息技术工程系}
{\CJKfamily{note} 泰山职业技术学院}
{\CJKfamily{new} 华文中宋 华文中宋}
信息技术工程系

\setCJKfamilyfont{xiaobi}{方正小标宋简体}
{\CJKfamily{xiaobi}信息技术工程系}
黑体 信息技术工程系
\begin{note}

\hfill 信 息 技 术 工 程 系
\end{note}
\end{document}
