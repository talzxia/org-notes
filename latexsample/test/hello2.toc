\contentsline {chapter}{\numberline {1}泰山职业技术学院}{1}{chapter.1}
\contentsline {section}{\numberline {1.1}信息技术工程系}{1}{section.1.1}
