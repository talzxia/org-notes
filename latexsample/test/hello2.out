\BOOKMARK [0][]{chapter.1}{1 泰山职业技术学院}{}% 1
\BOOKMARK [1][]{section.1.1}{1.1 信息技术工程系}{chapter.1}% 2
